packages/fast-components-react-base/src/checkbox/README.md

```diff
@@ -0,0 +1,12 @@
+# Implementation
```

Comment:
@chrisdholt Do we need to copy over this README.md as well?

Reply:
I think we can for now